lib/config/default-scopes.js

```diff
@@ -7,9 +7,10 @@ export default [
       'import\\s+.*?from\\s+[\'"]', // import foo from './foo'
       'import\s+[\'"]', // import './foo'
       'require\\([\'"]', // require('./foo')
-      'define\\(\\[?[\'"]' // define(['./foo']) or define('./foo')
+      'define\\(\\[?[\'"]', // define(['./foo']) or define('./foo')
+      '@import[\\(|\\s+]?[\'"]', // @import 'foo' or @import('foo')
     ],
-    extensions: ['js', 'jsx', 'vue', 'ts', 'tsx', 'coffee'],
+    extensions: ['js', 'jsx', 'vue', 'ts', 'tsx', 'coffee','sass', 'scss', 'css', 'less'],
```

Comment:
Why you didn't add extensions for Stylus files? Default boilerplate from vue-cli supports it right out-of-box.

Reply:
Sorry, I forgot about it :(